parent/pom.xml

```diff
@@ -297,6 +300,12 @@
         <version>${version.grpc}</version>
       </dependency>
 
+      <dependency>
+        <groupId>io.grpc</groupId>
+        <artifactId>grpc-protobuf-lite</artifactId>
```

Comment:
Do we need that?

Reply:
Yes as the new dependency depends on an old version of `grpc-protobuf-lite`, which then leads to a dependency convergence error during build.

Reply:
Ok thanks